packages/form/src/Slider.ts

```diff
@@ -91,18 +93,60 @@ export class Slider extends SVGWidget {
         this.slider
             .attr("transform", "translate(" + (-this.width() / 2 + this.padding()) + "," + 0 + ")");
 
-        this.slider.selectAll("line")
+        this.slider.selectAll("line.track,line.track-inset,line.track-overlay")
             .attr("x1", this.xScale.range()[0])
             .attr("x2", this.xScale.range()[1])
             ;
+        let tickText;
+        const x_distance = (this.width() - (this.padding() * 2)) / (this.tickCount() - 1);
+        if (this.tickDateFormat() !== null) {
+            const Q_parser = d3TimeParse("%Q");
```

Comment:
Should this be using `dateParseFormat()`?

Reply:
This is something I wanted to discuss with you. Dates are currently expected as milliseconds since epoch and then formatted with *tickDateFormat()*. It makes the tick datetimes much easier to work with.